iOSNativeAotApp/iOSNativeAotApp.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)-ios</TargetFramework>
+    <EnablePreviewFeatures>True</EnablePreviewFeatures>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+    <RuntimeIdentifiers>ios-arm64</RuntimeIdentifiers>
+    <OutputType>Exe</OutputType>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>true</ImplicitUsings>
+    <SupportedOSPlatformVersion>11.0</SupportedOSPlatformVersion>
+    <PublishAot>true</PublishAot>
+    <_IsPublishing>true</_IsPublishing>
```

Comment:
per the docs here: https://github.com/xamarin/xamarin-macios/blob/main/docs/nativeaot.md

added the `_IsPublishing` property to follow the precedent set by the other sample projects to continue using `dotnet build` for submitting to apple